js/jquery-comments.js

```diff
@@ -594,13 +594,23 @@
         },
 
         updateToggleAllButton: function(parentEl) {
+            // Don't hide replies if maxRepliesVisible has negative value
+            if (this.options.maxRepliesVisible < 0) return;
```

Comment:
Good point :) But I still think it's more logical to check if the value is null instead of a negative number